drivers/adc/ad738x/ad738x.c

```diff
@@ -44,7 +44,9 @@
 #include "stdio.h"
 #include "stdlib.h"
 #include "stdbool.h"
+#if !defined(USE_STANDARD_SPI)
```

Comment:
I think standard SPI should be the default code usage of the driver.
I'd do something like:
`if defined(USE_SPI_ENGINE)` insead.

Reply:
This make sense to me as well, however the original drivers were created to use with SPI engine, so retained that default functionality. The same thing was done for ad4696. I am okay with modifying this code to use standard SPI as default mode, so just let me know your inputs and I can modify it accordingly.